reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md

```diff
@@ -0,0 +1,345 @@
+# What's New in PowerShell Core 6.0
+
+## Renamed `powershell(.exe)` to `pwsh(.exe)`
+
+The binary name for PowerShell Core has been changed from `powershell(.exe)` to `pwsh(.exe)` in order to provide a deterministic way for users to open PowerShell Core on machines where both Windows PowerShell and PowerShell Core are installed.
+
+`pwsh` is also much shorter and easier to type.
+
+Additionally, we've made some changes to `pwsh(.exe)`:
+
+- Change positional parameter for `powershell.exe` from `-Command` to `-File`.
+ This fixes the usage of `#!` (aka as a shebang) in PowerShell scripts that are being executed from non-PowerShell shells on non-Windows platforms.
+ This also means that you can now do things like `powershell foo.ps1` or `powershell fooScript` without specifying `-File`.
+ However, this change now requires that you explicitly specify `-c` or `-Command` when trying to do things like `powershell.exe Get-Command`.
+ (#4019)
+- Update PowerShell Core to accept the `-i` (or `-Interactive`) switch to indicate an interactive shell. (#3558)
+ This will help when using PowerShell as a default shell on Unix platforms.
+- Remove parameters `-importsystemmodules` and `-psconsoleFile` from `pwsh.exe`. (#4995)
+- Fix `pwsh -version` and built-in help for `pwsh.exe` to align with other native tools. (#4958 & #4931) (Thanks @iSazonov)
+- Make invalid argument error messages for `-File` and `-Command` consistent and make exit codes consistent with Unix standards (#4573)
+- Add `-WindowStyle` parameter on Windows. (#4573)
+
+## Moved from .NET Framework to .NET Core
+
+TODO
+
+## Support for macOS and Linux
+
+PowerShell now supports macOS and Linux, including:
+
+TODO
+
+We also made a number of changes to PowerShell in order to make it work better with non-Windows ecosystems.
+Some of these are breaking changes which also affect Windows.
+Others are only present or applicable in non-Windows installations of PowerShell Core.
+
+TODO: organize
+- Add support for native command globbing on Unix platforms.
+- `more` function respects the Linux `$PAGER`; defaults to `less`
+ This means you can now use wildcards with native binaries/commands (e.g. `ls *.txt`). (#3463)
+- Escape trailing backslash when dealing with native command arguments. (#4965)
+- Ignore the `-ExecutionPolicy` switch when running PowerShell on non-Windows platforms because script signing is not currently supported. (#3481)
+- Fix ConsoleHost to honour `NoEcho` on Unix platforms. (#3801)
+- Fix `Get-Help` to support case insensitive pattern matching on Unix platforms. (#3852)
+- `powershell` man-page added to package
+
+### Logging
```

Comment:
Might be worth mentioning that on Windows, we have our own PowerShell channel which is distinct from Windows PowerShell